src/Components/PageNavbar_Components/NavModal.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
```

Comment:
Please give this a better name. This modal does not by definition have anything to do with the nav. That's just the place we happen to trigger it from